packages/docs/src/routes/(ecosystem)/media/index.tsx

```diff
@@ -391,22 +517,44 @@ export const MEDIA = mediaObj({
 export interface MediaEntry {
   title: string;
   href: string;
+  author?: string;
   language?: string;
+  country?: string;
```

Comment:
i am not sure if a country has an impact when a user wants to view a video. what was the idea behind this?

Reply:
i don't know which variable name should i put because some of the video title was prefixed with country flag so i assumed it would be better and it shows the language's country

Reply:
thx a lot for the quick reply 🙏 imo just showing the language would be sufficient. the goal of the flag in front of the title is just to indicate in which language smth was recorded 👼  would you mind to remove it again?

Reply:
should i remove the country the type from the MediaObj Interface, put the flag in front of title and i will put the language only.

Reply:
imo just having the language badge would be sufficient. guess the flag could be left out then 🙏 thx again